### DISCUSSION TITLE
How to optimize ram memory?

### DISCUSSION THREAD

**Discussion by manhtam922004** [engageub/InternetIncome#86] [Q&A]
Hello engageub and everyone!
I run 150 docker with proxy, ram memory takes up 10gb, and increases over time.
I run a only service is repocket, so I want to reduce ram memory.
Thank you so much engageub and everyone!

**Comment by engageub** [comment]
Hello,
Could you please check your docker stats after starting the containers and then compare it with stats after an hour and see which containers are increasing memory gradually.
Most of the docker images are official docker images. If there is some issue which causes memory to be gradually increased it has to be followed up with the respective docker image owner.

```
sudo docker stats
```

Also please mention the following
1. OS Used
2. Architecture whether is is amd64 or arm64
3. Are logs disabled in properties.conf


Using the above, you will know which container is gradually increasing and does not stop at a particular memory limit.
If you wish to limit the cpu and memory, you can use test branch where this option for limiting on each container can be specified. 

You can also increase your swapspace if you do not have sufficient RAM available on your linux.

Thank you
